extension/parquet/parquet_timestamp.cpp

```diff
@@ -105,6 +105,16 @@ date_t ParquetIntToDate(const int32_t &raw_date) {
 	return date_t(raw_date);
 }
 
+template <typename T>
+static T ParquetWrapTime(const T &raw, const T day) {
+	// Special case 24:00:00
+	if (raw == day) {
+		return raw;
+	}
+	const auto modulus = raw % day;
```

Comment:
Instead of dong the modulo should we perhaps throw an error here? That is what pandas seems to do:

```py
pd.read_parquet('timetz_column.parquet')
# ValueError: hour must be in 0..23
```